### PR TITLE
EE-2364 - new sample and individual attributes

### PR DESCRIPTION
**Pull request by M-casado**
## Ticket reference
EE-2364
## Overall changes
- [x] Added new attributes to samples and individuals (from spreadsheet)
- [x] Tried to align with Phenopackets without overlapping information 
- [x] Transformed sample into an object that can be distributed publicly following EGA logic, while most sensitive metadata is contained within the individual and future phenopacket. 
- [x] Added examples with those new or modified attributes 

## Future TO-DOs
- [ ] Add to CHANGELOG

### CODE REVIEW

**Review by Dona094 — APPROVED** [review]
Have checked the PR 👍 Thank you for preparing the spreadsheet, it was easier to see the proposed attributes this way. All the proposed attributes look good to me. Verified that the three mandatory fields (subject_id, phenotype, and sex)  are now in individual. The phenotype and disease descriptors of Type: Array looks like a great addition as well as the sample_status addition

File: schemas/EGA.individual.json [hunk 1]
```diff
@@ -41,20 +41,47 @@
       "minimal_public_attributes":{
         "type": "object",
         "title": "Minimal public attributes describing an individual",
-        "description": "Among all attributes describing an individual, some may contain identifiable metadata and thus must be private. Nevertheless, there are three required attributes (even if they are unknown): subject id, biological sex and phenotype. These shall be displayed and queryable.",
+        "description": "Among all attributes describing an individual, some may contain identifiable metadata and thus must be private. Nevertheless, there are three/four required attributes (even if they are unknown): subject id, biological sex and phenotype. These shall be displayed and queryable. In the case of a healthy individual (with no phenotypic abnormalities nor diseases), the 'phenotypes' and 'diseases' arrays will contain a reference to 'Unaffected' [NCIT:C94232].",
         "additionalProperties": false,
-        "required": ["subject_id", "biological_sex", "phenotype"],
+        "required": ["subject_id", "biological_sex"],
```
Comment (silviabah) [inline schemas/EGA.individual.json:46]: Just double checking :)  In the description in the ticket, it is mentioned that "subject_id, phenotype and sex" are mandatory fields, but phenotype is not included in required.

Comment (silviabah) [reply schemas/EGA.individual.json:46]: I found them below :)

**Review by silviabah — APPROVED** [review]
All changes mentioned in the ticket description have been implemented.

**Pull request merged**